### ISSUE TITLE
Requisitos/Funcionalidades Iniciais

### ISSUE THREAD

**Issue by tuliofaria** [softwaredobem/boraajudar#2]
Pessoal, nesta issue comentem funcionalidades que vocês julguem importantes para o projeto BoraAjudar - seguindo o propósito do mesmo: 

> Ser uma ferramenta para Voluntários e Instituições filantrópicas se encontrarem e ajudarem.

**Comment by sandrooliveira** [comment]
Pensei nas funcionalidades abaixo:

- Serviço de Login

- Cadastro de instituições
  Cada instituição pode se cadastrar e criar seu perfil

- Cadastro de voluntários
 
- Pesquisa de instituições com filtros por cidade, tipo etc.

- Quando uma nova instituição se cadastrar emitir uma notificação a todos os voluntários

- Doações
  Um voluntário pode doar livremente a uma instituição que deseja ou a instituição pode postar em seu perfil um projeto ou uma necessidade específica a qual seja preciso angariar recursos e o voluntário pode doar diretamente para aquele fim. Ainda para cada projeto/necessidade pode haver uma meta como é feito no site https://www.vakinha.com.br/.
 
- Timeline com necessidades/projetos/resultados das instituições que o voluntário segue.
   Neste ponto seria copiar +/- a ideia do Facebook, então não sei se isso daria algum problema com a licença do React mas a ideia é que um voluntário poderia seguir uma ou mais instituições e estas poderiam divulgar seus projetos e necessidades ou postar resultados obtidos através da ajuda dos voluntários e agradecimentos em sua timeline. Porém deveria haver uma regra para que somente as instituições postassem para não virar algo igual o grupo de Whatsapp com vídeos de zueira e imagens de bom/boa dia/tarde/noite, seria talvez mais a idea de uma newsletter das instituições.

**Comment by Patrick-Isidoro** [comment]
Gostei da ideia do @sandrooliveira muito boa, gostaria de complementar..
Acho que seria legal colocar tipo uma área de atuação para a instituição e talvez uma área proposta para o voluntário, facilitaria o filtro nas buscas.
Seria tipo:
Instituições -> Área de atuação, escolher por exemplo: creche, asilo, ou mais generalizado, tipo: atua com crianças, com idosos, com moradores de rua.. Algo que defina o propósito da Instituição.
Voluntários -> Seria o mesmo que para as Instituições, podendo selecionar que seja notificado somente quando Instituições que se encaixem no seu perfil estiverem procurando voluntários.

Acho que seria interessante também, elaborar os termos de uso e tals... Para poder assegurar ambos os lados (Instituições e Voluntários), e não haver "bagunça" no projeto, como disse o Sandro a exemplo dos grupos no Whats.

**Comment by darcyjunior** [comment]
Concordo com todos os tópicos abordados pele @sandrooliveira e pelo @Patrick-Isidoro, e vem muito de encontro com as informações que eu coletei com um amigo que trabalha em uma casa de recuperação. 

Abaixo alguns detalhes que observei:

- A parte da validação da instituição é bem importante para dar credibilidade ao site/projeto. Pois infelizmente há muitos aproveitadores.

- Talvez ter no perfil da instituição uma área para contar um pouco da história de cada uma.

- A ideia de uma área onde as instituições possam mostrar os resultados é muito relevante. Seria uma forma de prestação de contas do auxílio que estão recebendo. Por exemplo na instituição que ajudo sempre eles mandam um status dos projetos que eles solicitam recursos. A imagem abaixo foi enviada referente ao projeto da troca do telhado que a muito tempo estava causando infiltrações e etc.
![img-20170125-wa0025](https://user-images.githubusercontent.com/25440250/30004472-102e2702-90a6-11e7-804a-dc9247467fca.jpg)

Do mais assim que tiver mais ideias e informações posto aqui.

**Comment by wdmoreira** [comment]
Muito interessante as funcionalidades propostas. 
 - Como seriam feitas as validações das instituições?
 - A área para mostrar os resultados, seria posters com fotos, vídeos, amostra de produtos feitos pelas instituições, textos e prestação de contas das doações e uso delas?  
- Redes sociais, o software terá essa integração?

**Comment by batistafrancis** [comment]
Por hora, minha sugestão é listar as integrações e funcionalidades:

* Email marketing
* Push notifications
* Redes Sociais
* Encurtadores de url como bit.ly
* Agregadores de feed como reddit
* Algum serviço de XML-RPC (ou criarmos o nosso)

**Comment by tuliofaria** [comment]
Olá @wdmoreira , acredito que as validações por enquanto pode ser feita manualmente por todos nós. Baseado em alguns critérios. Por exemplo, envio de documentos, comprovantes bancários, etc. Apenas para nos garantir.

**Comment by gontijobr** [comment]
Bem, nessa relação @tuliofaria como venho de empresas com ERP com foco em Controladoria, toda a parte Fiscal/Contábil eu consigo direcionar os trabalhos, incluindo a parte Financeira e Faturamento (com NFS-e)
Agora se precisarem documentar processos pessoal, manda ae que eu faço as atividades. Caso queiram meu email é gontijobr@gmail.com e skype: gabrielrodriguesgontijo

**Comment by batistafrancis** [comment]
Como ficamos com este projeto?
Estive pensando nesta ideia durante esta pandemia.
Estou desenvolvendo um projeto para pequenos comércios da minha cidade e o **BoraAjudar** poderia ser incorporado nisso.

**Comment by tuliofaria** [comment]
@francisrod01 dá uma olhada depois em @id49/vaidarcerto.org , fizemos um visão inicial de ajuda também. Vamos movimentar mais o @softwaredobem em breve ;)
